lddtree.sh

```diff
@@ -24,6 +24,7 @@ usage() {
 	  -b              Force use of specific backend tools (scanelf or binutils)
 	  -R <root>       Use this ROOT filesystem tree
 	  --no-auto-root  Do not automatically prefix input ELFs with ROOT
+	  --no-recursive  Do not recursivly parse dependencies
```

Comment:
So we can parse output and focus only on direct linked lib.
usage example :
output=$(/lddtree/lddtree.sh --no-recursive --no-header $_path)